server/ws/ws_connect_handler.py

```diff
@@ -21,3 +21,5 @@ def disconnect():
     redis.delete(get_jwt_identity()["_id"])
     return
 
+# emit("sent_email_status", status_dict, room=room_id)
```

Comment:
Should these comments be removed, or do we need the code later?

Reply:
We have these code snippet somewhere else, this is just easy to see what event and data will be emitted